src/components/KYCWall/BaseKYCWall.js

```diff
@@ -97,11 +97,12 @@ class KYCWall extends React.Component {
     continue(event, iouPaymentType) {
         this.setState({transferBalanceButton: event.nativeEvent.target});
         const isExpenseReport = ReportUtils.isExpenseReport(this.props.iouReport);
+        const paymentCardList = this.props.fundList || this.props.cardList;
```

Comment:
```suggestion
        const paymentCardList = this.props.fundList || this.props.cardList || {};
```

Not sure if needed, but this might preserve the default empty object behavior we are changing in the case that both of these values are `null`.